code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -79,7 +79,7 @@
 
 	for(var/i in rad_places)
 		var/turf/place = i
-		if(get_dist(user, place) >= range * 5)	//Rads are easier to see than wires under the floor
+		if(range != -1 && get_dist(user, place) >= range)
```

Comment:
What's the reason for the `range != -1` check here?

Reply:
It's a special "ignore any range checks" value, essentially. I can't really think of a good reason it'd be necessary to have it larger than the screen size I suppose, and if there's a way to pull that var from the client it'd probably be a better option.

Reply:
I don't understand, because you're not doing `show_rads(-1)` anywhere to warrant including this. In the event that you do need this though, it should probably be placed before the loop, that way you don't have to check if range is -1 for every entry in the list.

Reply:
Yeah, I was mostly including it in case it might be useful down the line. I don't really see a use for it (since it'd probably not be worth looping over every single item when we have a range implemented) so I'll just remove it.

Reply:
Removed the check in 000b89a